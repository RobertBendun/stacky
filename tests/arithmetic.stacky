# dot compare
"io.stacky" include

25 10 +      .
25 10 -      .
25 10 *      .
25 10 div    .
25 10 mod    .
25 10 divmod . .
4 2 << .
8 2 >> .

20 10 bit-and .
20 10 bit-or  .
20 10 bit-xor .

30 40 min .
40 30 min .
30 40 max .
30 40 max .
