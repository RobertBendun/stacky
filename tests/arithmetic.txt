35
15
250
2
5
2
5
16
2
0
30
30
1
1
0
0
0
1
0
1
1
0
0
0
1
1
1
0
