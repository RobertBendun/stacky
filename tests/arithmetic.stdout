35
125
250
2
5
2
5
16
2
0
30
30
30
30
40
40
